### DISCUSSION TITLE
Is it possible to change linking keywords for issues?

### DISCUSSION THREAD

**Discussion by tracey-le** [community/community#38606] [Projects and Issues]
### Background

In pull requests, if you add a keyword (such as "closes", "fixes") before linking an issue, it will link that issue to that pull request.

(For more info: [Linking a pull request to an issue using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

### Question/Feature request
Is it possible to change what these keywords are, or add to the list?

Some people on my team use "finishes". And I imagine for people who may not speak English as first language, that they might have a preference for other words too.

**Comment by ossanna16** [comment]
Hi there @t1-tracey and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
